windows-driver-docs-pr/image/sample-inf-file-for-a-web-services-scanner.md

```diff
@@ -33,20 +33,14 @@ YourWIADriver.dll=1
 [SourceDisksNames.x86]
 1=%Location%,,,
 
-[SourceDisksFiles.ia64]
-YourWIADriver.dll=1
-
-[SourceDisksNames.ia64]
-1=%Location%,,,
-
 [SourceDisksFiles.amd64]
 YourWIADriver.dll=1
 
 [SourceDisksNames.amd64]
 1=%Location%,,,
 
 [DestinationDirs]
-DefaultDestDir = 11
+DefaultDestDir = 13
 
 [Manufacturer]
 %Mfg%=Models,NTx86,NTamd64,NTia64
```

Comment:
Since the SourceDisksFiles and SourceDisksNames are being removed for ia64, you should also remove the NTia64 match here and remove the [Models.NTia64] section below